pkg/kubelet/container/runtime.go

```diff
@@ -97,6 +97,8 @@ type Runtime interface {
 	RemoveImage(image ImageSpec) error
 	// Returns Image statistics.
 	ImageStats() (*ImageStats, error)
+	// Cadvisor FS Label
+	ImageLabel() string
```

Comment:
@sjpotter I know it's not runtime's name, I mean kubelet should be able to decide the label by itself when given the container runtime's name.
It just seems unnecessary to have runtime implements this.